SCons/Environment.xml

```diff
@@ -778,26 +778,35 @@ See also &f-link-env-AppendUnique;,
 </arguments>
 <summary>
 <para>
-Append path elements specified by <parameter>newpath</parameter>
-to the given search path string or list <parameter>name</parameter>
-in mapping <parameter>envname</parameter> in the &consenv;.
-Supplying <parameter>envname</parameter> is optional:
-the default is the execution environment &cv-link-ENV;.
-Optional <parameter>sep</parameter> is used as the search path separator,
-the default is the platform's separator (<systemitem>os.pathsep</systemitem>).
-A path element will only appear once.
-Any duplicates in <parameter>newpath</parameter> are dropped,
-keeping the last appearing (to preserve path order).
-If <parameter>delete_existing</parameter>
-is <constant>False</constant> (the default)
-any addition duplicating an existing path element is ignored;
-if <parameter>delete_existing</parameter>
-is <constant>True</constant> the existing value will
-be dropped and the path element will be added at the end.
-To help maintain uniqueness all paths are normalized (using
-<systemitem>os.path.normpath</systemitem>
-and
-<systemitem>os.path.normcase</systemitem>).
+Append one or more path prefixes to an entry in a dictionary-valued
```

Comment:
use "path" instead of "path prefixes"?

Reply:
We've never used the term before in SCons docs..  we've used the term as a prefix to a path, but not as adding it to a list of paths to search.

IMHO they're a collection of paths to be searched..

Here's what we've used in SCons docs.. 

When called as a construction environment method, searches the **paths** in the path keyword argument, or if None (the default) the **paths** listed in the construction environment (env['ENV']['PATH']). The external environment's **path** list (os.environ['PATH']) is used as a fallback if the key env['ENV']['PATH'] does not exist.

I find "path prefixes" to be more confusing than just paths or path.

Reply:
One would not expect the author of text to find it confusing, but rather feed back from the consumers of such would be able to point it out.. ?